confluent-cloud_kickstart/confluent-cloud_kickstart.py

```diff
@@ -95,9 +95,11 @@ def write_to_file(file_name, text, json_fmt=True):
 print("Enabling the API key for the Kafka cluster")
 cli(["confluent", "api-key", "use", creds_json['api_key'], "--resource", cluster_json['id']], debug, fmt_json=False)
```

Comment:
This flag is no longer required here and can be removed!
```suggestion
cli(["confluent", "api-key", "use", creds_json['api_key']], debug, fmt_json=False)
```

Reply:
whoops! I was too fast on the push, I'll do a follow-up PR now

Reply:
Hmm, I'm not sure, now that I think about it, you need to specify which resource you're generating keys for.
I'll test it out

Reply:
Doing that results in an error
```
Error: required flag(s) "resource" not set
```

Reply:
`--resource` is required for creating API keys, but no longer required for using them!